### ISSUE TITLE
使用自己数据集训练yolov3增强模型时报错

### ISSUE THREAD

**Issue by llzhaoshuo** [PaddlePaddle/PaddleDetection#806]
仿照了coco数据集的格式，修改了num_classes=2,命令如下：
python -u tools/train.py -c configs/dcn/yolov3_r50vd_dcn_db_iouaware_obj365_pretrained_coco.yml -o pretrain_weights=/home/ResNet50_vd_dcn_db_obj365_pretrained

报错信息如下：
----------------------
Error Message Summary:
----------------------
Error: The input's size along the split dimension must be evenly divisible by Attr(num_or_sections). But received Attr(num_or_sections) = 8, input(X)'s shape = [2, 50, 4], Attr(dim) = 0.

**Comment by heavengate** [comment]
是不是修改了`batch_size`呢，看着像是把reader里的`batch_size`修改成了4但是`YOLOv3Loss`里没有修改，可以确定一下`configs/dcn/yolov3_r50vd_dcn_db_iouaware_obj365_pretrained_coco.yml`里的`YOLOv3Loss.batch_size`和`configs/dcn/yolov3_enhance_reader.yml`里的`TrainReader.batch_size`是否一致

**Comment by llzhaoshuo** [comment]
> 是不是修改了`batch_size`呢，看着像是把reader里的`batch_size`修改成了4但是`YOLOv3Loss`里没有修改，可以确定一下`configs/dcn/yolov3_r50vd_dcn_db_iouaware_obj365_pretrained_coco.yml`里的`YOLOv3Loss.batch_size`和`configs/dcn/yolov3_enhance_reader.yml`里的`TrainReader.batch_size`是否一致

按照您说的，已解决，感谢

**Comment by qingqing01** [comment]
@llzhaoshuo 如果问题解决了，关下issue吧，后续会优化下参数设置和文档。
